lighthouse-core/audits/interactive.js

```diff
@@ -14,16 +14,15 @@ const Util = require('../report/html/renderer/util');
  * and returns the timestamp of the beginning of the CPU quiet period.
  * @see https://docs.google.com/document/d/1GGiI9-7KeY3TPqS3YT271upUVimo-XiL5mwWorDUD4c/edit#
  */
-class ConsistentlyInteractiveMetric extends Audit {
+class InteractiveMetric extends Audit {
   /**
    * @return {!AuditMeta}
    */
   static get meta() {
     return {
-      name: 'consistently-interactive',
-      description: 'Consistently Interactive (beta)',
-      helpText: 'Consistently Interactive marks the time at which the page is ' +
-          'fully interactive. ' +
+      name: 'interactive',
+      description: 'Interactive',
```

Comment:
IMO this should be called the full "Time to Interactive" in the UI. even if there's inconsistency, that's the name.

Reply:
done!